File: Relaciones/ejercicio 1/Empleado.java
package relaciones;
public class Empleado {
    private String  nombre,cargo;
    private double sueldo;
    
    public Empleado(){
        nombre="Gerson";
        cargo="Supervisor";
        sueldo=1200;
    }
    public Empleado(String n){
        nombre=n;
        cargo="gerente";
        sueldo=3000;
    } 
    public void mostrar(){
        System.out.println(nombre+" "+cargo+" "+sueldo);
    }
    public double getSueldo() {
        return sueldo;
    }
    public void setSueldo(double sueldo) {
        this.sueldo = sueldo;
    }
    public String getNombre() {
        return nombre;
    }
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
}

File: Relaciones/ejercicio 1/Empleado.py
class Empleado:
    def __init__(self, n=None):
        if n is None:
            self.nombre = "Gerson"
            self.cargo = "Supervisor"
            self.sueldo = 1200
        else:
            self.nombre = n
            self.cargo = "gerente"
            self.sueldo = 3000

    def mostrar(self):
        print(self.nombre, self.cargo, self.sueldo)

    def getSueldo(self):
        return self.sueldo

    def setSueldo(self, sueldo):
        self.sueldo = sueldo

    def getNombre(self):
        return self.nombre

    def setNombre(self, nombre):
        self.nombre = nombre